shared/src/main/scala/drt/shared/redlist/RedList.scala

```diff
@@ -0,0 +1,213 @@
+package drt.shared.redlist
+
+import drt.shared.CrunchApi.MillisSinceEpoch
+import drt.shared.Terminals.{T2, T3, T5, Terminal}
+import drt.shared.api.PassengerInfoSummary
+import drt.shared.{ApiFlightWithSplits, Nationality, PortCode}
+
+import scala.collection.immutable.Map
+
+object RedList {
+  val change20210808 = 1628377200000L
```

Comment:
is it only one specific date we are considering for red list ? I was thinking like every few weeks list is changing and are we not changing list according to it ?

Reply:
This change is to support the incoming red list changes. We can go back and implement the previous several changes, but that wasn't a requirement for this work. We just wanted the incoming changes on Sunday to be made without us having to make a deployment on Sunday morning

Reply:
Okay . So we have to do this for every time new changes is comes alone and change the date .